db/migrate/20210609041826_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :firstname, :string
```

Comment:
In Rails, we prefer to use snake case while naming
```suggestion
    add_column :users, :first_name, :string
```

Reply:
yes, more an English issue from me than a rails convention issue here ˆˆ, 
I've renamed it (db migration added + model refactor).